index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <div class="container">
+      <header>
+        <h1 class="logo">Welcome</h1>
+        <a href="#"><img src="images/menu_icon.png" alt="menu" /></a>
+      </header>
+
+      <section class="headline">
+        <h2>
+          Hey there. I’m Mariane <br /><span>I’m a software developer</span>
+        </h2>
```

Comment:
- It would be better to use `h1` instead of `h2` tag for the main title to improve our structure and semantics. In your code `Hey there. I’m Mariane <br /><span>I’m a software developer</span>`  is the main title of your page.